packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/contact-section/contact-section.ejs

```diff
@@ -0,0 +1,117 @@
+<%_ if (profile.relevantLinks && profile.relevantLinks.length > 0 || profile.phoneNumbers && profile.phoneNumbers.length > 0 ||profile.emails && profile.emails.length > 0 || profile.city && profile.country ) { -%>
```

Comment:
This is wrong. You can get the country but not the city. Replace by:

`<%_ if (profile.relevantLinks && profile.relevantLinks.length > 0 || profile.emails && profile.emails.length > 0 || profile.city || profile.country || profile.phoneNumbers && profile.phoneNumbers.length > 0) { -%>`